tools_tag_converters/Convertor-Stubbs_Coll_to_Halo_Coll.py

```diff
@@ -123,7 +130,7 @@ def __init__(self, *args, **kwargs):
         self.resizable(0, 0)
 
         self.tags_dir = StringVar(self)
-        self.tags_dir.set(curr_dir + 'tags' + PATHDIV)
+        self.tags_dir.set(curr_dir + 'tags/')
```

Comment:
What's the point of always appending the '/'? Is it so you can avoid using path.join()?

What happens if our starting path has backslashes?

Reply:
As I stated above to csauve, the slash can be removed. This is a remnant of old habits dying hard.

Reply:
Could we handle that?

Reply:
done. also, just tested all converters on windows and opened converted tags in mozz. tags load and appear fine.